src/Files.App/UserControls/Widgets/FolderWidget.xaml

```diff
@@ -8,6 +8,23 @@
 	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 	Background="{ThemeResource SolidBackgroundFillColorBaseBrush}"
 	mc:Ignorable="d">
+
+	<UserControl.Resources>
```

Comment:
What does this button style over the existing style?

Reply:
I don't know what do you mean. If you mean what's the benefits to apply this style, I'd say the reasons are the same as the reasons why Microsoft Store does use some kind of this style(I call enhancement effect - make a different from others). And I think the default button style is not suitable for this, Widget items.

Pointer over on felt-bottom item

![image](https://user-images.githubusercontent.com/62196528/216055964-514a6ecd-5c5c-40cf-94fd-c271fc2cddea.png)